karma.conf.js

```diff
@@ -37,6 +37,10 @@ module.exports = () => {
       },
     ],
     customLaunchers: {
+      ChromeDesktop: {
+        base: 'Chrome',
+        flags: ['--window-size=1920,1080'],
```

Comment:
This is added to address the issues we were seeing with the `SkyScrollShadowDirective` tests. If you want to run the debugger (watch mode) locally, run `npx nx test core --browsers=ChromeDesktop`.